### ISSUE TITLE
Twython cursor and get_user_timeline doesn't know when to stop

### ISSUE THREAD

**Issue by chasrmartin** [ryanmcgrath/twython#365]
```
    timeline = []
    results = twitter.cursor(
        twitter.get_user_timeline
        , screen_name=customer_handle
        , count=200
        , include_entities=True
        , include_rts=1
    )
    count = 0
    for result in results:
        logger.debug("Got result, id: %s" % result['id_str'])
        count += 1
        if count > 1000: break
        timeline.append(result)
```

Here's some selected results:

```
retrieve_customer_timeline[77]: Got result, id: 524280237828231168
retrieve_customer_timeline[77]: Got result, id: 523492215642152960
retrieve_customer_timeline[77]: Got result, id: 523147764754956288
retrieve_customer_timeline[77]: Got result, id: 522931696573091841
retrieve_customer_timeline[77]: Got result, id: 522925316201644032
retrieve_customer_timeline[77]: Got result, id: 522862141448011776
retrieve_customer_timeline[77]: Got result, id: 522857627722985473
retrieve_customer_timeline[77]: Got result, id: 522823556078391296
retrieve_customer_timeline[77]: Got result, id: 522810925347123200
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
retrieve_customer_timeline[77]: Got result, id: 543084740492804096
```

**Comment by michaelhelmick** [comment]
When are you expecting results to stop?

**Comment by chasrmartin** [comment]
After it retrieves a different bunch of results.  If you observe above, it's repeatedly getting the *same* 200 messages over and over, never advancing through the timeline.

**Comment by mandarvaze** [comment]
Duplicate of #325 ?

**Comment by ryanmcgrath** [comment]
Hmmm, this seems like it's been sitting for a bit. @michaelhelmick, am I stepping on your toes if I look into merging it this weekend? Seems like it's still a bug.

**Comment by michaelhelmick** [comment]
@ryanmcgrath be my guest. I wanted to cut a release by the end of this month. I just have a work deadline this week. So I'll have some time after that!

**Comment by michaelhelmick** [comment]
@chasrmartin Sorry for getting to this so late. Can you try @eoso PR (#386) and let me know if that solves the issue? I'll merge if so!

**Comment by RedactedCode** [comment]
Still experiencing this problem 
Python2.7 on Linux.
The mentioned fix doesn't seem to have made it's way into the live release yet

**Comment by michaelhelmick** [comment]
@RedactedCode it hasn't, waiting on @eoso to update the Pull Request
